README.md

```diff
@@ -37,6 +37,18 @@ The package is [automatically linked](https://github.com/react-native-community/
 npx pod-install
 ```
 
+## Fabric
```

Comment:
@SudoPlz 
I believe this part of the readme is not necessary as the podfile automatically includes those headers when ` :fabric_enabled(:fabric_enabled => true)`. What do you think?

Reply:
that might be true, we need to double check. On my project I actually needed to add those to my Podfile because the `modular_headers` option wasn't added automatically.

For other projects this may not be required, but I guess it depends on how the Podfile of the consuming iOS project is structured.